tests/phpunit/Configuration/Entry/PhpUnitTest.php

```diff
@@ -43,44 +43,54 @@
 #[CoversClass(PhpUnit::class)]
 final class PhpUnitTest extends TestCase
 {
-    use PhpUnitAssertions;
-
-    #[DataProvider('valuesProvider')]
-    public function test_it_can_be_instantiated(
-        ?string $configDir,
-        ?string $executablePath,
+    #[DataProvider('basePathProvider')]
+    public function test_it_can_create_a_new_instance_with_absolute_paths(
+        PhpUnit $phpunit,
+        string $basePath,
+        PhpUnit $expected,
     ): void {
-        $phpUnit = new PhpUnit($configDir, $executablePath);
-
-        $this->assertPhpUnitStateIs($phpUnit, $configDir, $executablePath);
-    }
+        $originalPhpunit = clone $phpunit;
 
-    public function test_it_can_change_its_configuration_dir(): void
-    {
-        $phpUnit = new PhpUnit(
-            '/path/to/phpunit-config',
-            '/path/to/phpunit',
-        );
+        $actual = $phpunit->withAbsolutePaths($basePath);
 
-        $phpUnit->withConfigDir('/path/to/another-phpunit-config');
-
-        $this->assertPhpUnitStateIs(
-            $phpUnit,
-            '/path/to/another-phpunit-config',
-            '/path/to/phpunit',
-        );
+        $this->assertSame($expected, $actual);
```

Comment:
This is incorrect and due to PHP-CS-Fixer, see https://github.com/infection/infection/pull/2486